list.go

```diff
@@ -61,3 +61,23 @@ type ListsTweetsLookUpByIDResponse struct {
 	Detail   string              `json:"detail,omitempty"`
 	Type     string              `json:"type,omitempty"`
 }
+
+type ListFollowersLookUpByIDResponse struct {
+	Users    []*User             `json:"data"`
+	Includes *ListIncludes       `json:"includes,omitempty"`
+	Errors   []*APIResponseError `json:"errors,omitempty"`
+	Meta     *ListMeta           `json:"meta"`
+	Title    string              `json:"title,omitempty"`
```

Comment:
Please running following command and return this response.
`curl "https://api.twitter.com/2/lists/1111111111111111111111111111111/followers" -H "Authorization: Bearer $Token"`